src/components/templates/BoardBestTemplate.tsx

```diff
@@ -0,0 +1,102 @@
+import BestNotice from '../organisms/BestNotice';
+import BestMoreButton from '../atoms/BestMoreButton';
+import { useState } from 'react';
+import BestTypeButton from '../atoms/BestTypeButton';
+import { Stack } from '@/components/atoms/base';
+
+export interface pageViewListComponent {
+  key: number;
+  title: string;
+  stageKey: string;
+}
+
+export const pageViewList: Array<pageViewListComponent> = [
+  {
+    key: 0,
+    title: '실시간',
+    stageKey: 'RECENT',
+  },
+  {
+    key: 1,
+    title: '주간',
+    stageKey: 'LAST_WEEK',
+  },
+  {
+    key: 2,
+    title: '월간',
+    stageKey: 'LAST_MONTH',
+  },
+  {
+    key: 3,
+    title: '댓글순',
+    stageKey: 'REPLY_COUNT',
+  },
+  {
+    key: 4,
+    title: '추천순',
+    stageKey: 'VOTE_COUNT',
+  },
+];
+
+const BoardBestTemplate = ({ pageView }: { pageView: string }) => {
```

Comment:
```suggestion
const BoardBestTemplate = ({ pageView }: { pageView: 'RECENT' | 'LAST_WEEK' ... }) => {
```

정해진 값만 사용해야하니 리터럴로 수정해주세요~